src/openforms/emails/templatetags/appointments.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+
+from openforms.appointments.utils import get_client
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def appointment_information(context):
+    client = get_client()
+    return client.get_appointment_details_html(context["_appointment_id"])
```

Comment:
I suspect this will crash if there's no appointment ID in the template context.

Reply:
It will.  This function requires an appointment id and if there is no appointment id (either because one was not created or because it just wanted added to the context) then this shouldn't be called so I though it was better to crash than fail silently.

Reply:
Considering that the content editors/form designers are not experts on the DTL and the forgiveness typically found in the DTL (empty variable -> just output empty string), I think we should give some leeway here. If the context var is not present, just return an empty string.